test/functional/page_objects/header_page.js

```diff
@@ -256,6 +256,29 @@ export function HeaderPageProvider({ getService, getPageObjects }) {
     async isSharedTimefilterEnabled() {
       return await find.existsByCssSelector('[shared-timefilter=true]');
     }
+
+    async typeIntoReactSelect(testSubj, value) {
+      const select = await testSubjects.find(testSubj);
+      const input = await select.findByClassName('ui-select-search');
+      await input.type(value);
+      const activeSelection = await select.findByCssSelector('.ui-select-choices-row.active');
+      await activeSelection.click();
+    }
+
+    async addFilter(field, operator, values) {
```

Comment:
I think you can use the filterBar service for this.  See https://github.com/elastic/kibana/blob/master/test/functional/services/filter_bar.js#L26